script/helperFunctions.sh

```
@@ -1441,74 +1434,66 @@ function verifyContract() {
+  elif [ "$API_KEY" != "NO_ETHERSCAN_API_KEY_REQUIRED" ]; then
+    # make sure API key is not empty
+    if [ -z "$API_KEY" ]; then
+      echo "Error: Could not find API key for network $NETWORK"
+      return 1
     fi
+    # add API key to verification command
+    VERIFY_CMD="$VERIFY_CMD -e ${!API_KEY}"
+  fi
+
+  # Attempt verification with retries
+  while [ $COMMAND_STATUS -ne 0 -a $RETRY_COUNT -lt $MAX_RETRIES ]; do
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)